pkg/service/test/test.go

```diff
@@ -220,6 +220,10 @@ func (t *tester) Test(path string, testReportPath string, appCmd string, options
 		t.logger.Error("failed to initialise the test", zap.Error(err))
 		return false
 	}
+	// Check if no testset found
+	if len(initialisedValues.Sessions) == 0 {
+		t.logger.Info("no testcases found. Use record mode to record testcases.")
+	}
```

Comment:
program should just log and continue. it shouldn't give error if no test sets are found.

Reply:
But if you don't return something from here the whole code which written is below this will get executed right..?

Reply:
Which is not prefered We will exit test command as soon as we find that there are no tests right

Reply:
Yes code below condition will execute and wont exit the program.
I had this discussion on stack https://keploy.slack.com/archives/C03NWV6D02G/p1704735008385619

Reply:
I don't see a reason in the discussion(slack) why should the below code execute. According to me it should just return from there.

Reply:
Well continuing below code will print all the info logs regarding stopping of server,proxy. and
`test` command is successfully finished as a process even without testcases. so i guess just logging is fine.

Reply:
And if we exit when no testcases are found , it should be done in cmd/root.go itself (as a pre check). as it should not create all resources for test.

Reply:
We can't say which command is going to be executed when we are in root.go right..? lets says if record is executed then also we go through root and then reach record cmd right. If there is a way to configure it in the root.go that when test cmd is executed we should check the folder first and then execute cmd you can implement it. But I prefer this logic to be in cmd/test.go rather than service/test/test.go or cmd/root.go.

Reply:
Yeah this logic in cmd/test.go will be good. 
I will make these changes. Should I make new PR and close this?

Reply:
you can do it in the same PR